src/js/service/ImportService.js

```diff
@@ -130,4 +130,68 @@
       return pskl.utils.FrameUtils.createFromImage(resizedImage);
     });
   };
+
+  /**
+   * @param {!pskl.model.Piskel} originalPiskel
+   * @param {!pskl.model.Piskel} additionalPiskel
+   * @returns {pskl.model.Piskel}
+   */
+  // Adds frames from an additionalPiskel to an originalPiskel, adjusting the size of
+  // both piskels to the largest dimensions of either piskel, and centering the content.
+  // This will only work when there is only one layer.
+  // Multi layer animations are disabled from the UI in code-dot-org/piskel
+  ns.ImportService.prototype.mergePiskels = function(originalPiskel, additionalPiskel) {
```

Comment:
While you're updating the comment, could also do the following:

* Add `@throws {Error} if a Piskel with more than one layer is provided.`
* Call out that both `originalPiskel` and `additionalPiskel` are mutated by this call - that's a surprising property just reading the comment and signature right now.
* Could also mention that the returned Piskel is `originalPiskel`.